### PR TITLE
2019/kw14/fixed_windows_build_docu_fixed_server_start_text

### PR DESCRIPTION
**Pull request by Tirokk**
> [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-01T14:37:59Z" title="Monday, April 1st 2019, 4:37:59 pm +02:00">Apr 1, 2019</time>_
_Merged <time datetime="2019-04-01T17:17:27Z" title="Monday, April 1st 2019, 7:17:27 pm +02:00">Apr 1, 2019</time>_
---

Fixed windows build, updated readme (prerequisites & requirements), fixed backend server start log output to reflect proper values

Wrong use of `$npm_config` - see here:
> https://docs.npmjs.com/files/package.json#config
> https://docs.npmjs.com/misc/config
> https://docs.npmjs.com/misc/config#per-package-config-settings
> https://stackoverflow.com/questions/34027400/npm-scripts-config-variables-and-command-substitution-not-working-in-package-j

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -63,6 +70,17 @@ docker-compose down -v
...
 
+Download [Neo4j Community Edition](https://neo4j.com/download-center/#releases) and unpack the files.
+
+Download [Neo4j Apoc](https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases) and drop the file into the `plugins` folder of the just extracted Neo4j-Server
+
+Start Neo4j
+```
+neo4j\bin\neo4j start
+```
+and confirm it's running [here](http://localhost:7474)
```
Comment (Tirokk) [inline backend/README.md:81]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-01T14:55:52Z" title="Monday, April 1st 2019, 4:55:52 pm +02:00">Apr 1, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -63,6 +70,17 @@ docker-compose down -v
 
 Install dependencies:
 
+Download [Neo4j Enterprise Edition](https://neo4j.com/download-center/#releases) and unpack the files.
+
+Download [Neo4j Apoc](https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases) and drop the file into the `plugins` folder of the just extratced Neo4j-Server
+
+Start Neo4j
+```
+neo4j\bin\neo4j start
+```
+and confirm it's running [here](http://localhost:7687)
```


```suggestion
and confirm it's running [here](http://localhost:7474)
```

File: backend/README.md [hunk 2]
```diff
@@ -63,6 +70,17 @@ docker-compose down -v
 
 Install dependencies:
 
+Download [Neo4j Community Edition](https://neo4j.com/download-center/#releases) and unpack the files.
+
+Download [Neo4j Apoc](https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases) and drop the file into the `plugins` folder of the just extracted Neo4j-Server
```
Comment (Tirokk) [inline backend/README.md:75]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-01T14:56:24Z" title="Monday, April 1st 2019, 4:56:24 pm +02:00">Apr 1, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -63,6 +70,17 @@ docker-compose down -v
 
 Install dependencies:
 
+Download [Neo4j Enterprise Edition](https://neo4j.com/download-center/#releases) and unpack the files.
+
+Download [Neo4j Apoc](https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases) and drop the file into the `plugins` folder of the just extratced Neo4j-Server
```


```suggestion
Download [Neo4j Apoc](https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases) and drop the file into the `plugins` folder of the just extracted Neo4j-Server
```

File: backend/README.md [hunk 3]
```diff
@@ -63,6 +70,17 @@ docker-compose down -v
 
 Install dependencies:
 
+Download [Neo4j Community Edition](https://neo4j.com/download-center/#releases) and unpack the files.
```
Comment (Tirokk) [inline backend/README.md:73]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-01T14:56:44Z" title="Monday, April 1st 2019, 4:56:44 pm +02:00">Apr 1, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -63,6 +70,17 @@ docker-compose down -v
 
 Install dependencies:
 
+Download [Neo4j Enterprise Edition](https://neo4j.com/download-center/#releases) and unpack the files.
```


```suggestion
Download [Neo4j Community Edition](https://neo4j.com/download-center/#releases) and unpack the files.
```

**Comment by Tirokk** [comment]
> [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-01T15:04:55Z" title="Monday, April 1st 2019, 5:04:55 pm +02:00">Apr 1, 2019</time>_

---


> Here are some changes I would like to see, could you also update https://github.com/Human-Connection/Nitro-Documentation ?

Please merge it into the repo first -> https://github.com/Human-Connection/Human-Connection/issues/342

**Comment by Tirokk** [comment]
> [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-01T16:58:58Z" title="Monday, April 1st 2019, 6:58:58 pm +02:00">Apr 1, 2019</time>_

---


@ulfgebhardt I restarted the build. If it passes, feel free to hit the merge button.

**Review by Mogge — CHANGES_REQUESTED** [review]
Here are some changes I would like to see, could you also update https://github.com/Human-Connection/Nitro-Documentation ?

**Review by Mogge — APPROVED** [review]
Fair enough @ulfgebhardt :handshake:

**Pull request closed without merging**